utils/modeler/modeler.go

```diff
@@ -59,9 +68,9 @@ func (m *Modeler) mapToModel(data map[string]string, context string, rv reflect.
 			// We're nested... use some recursion...
```

Comment:
If this is a `*int` or something, that is going to fail, no? This seems to assume that if it's a PTR, it's pointing to a struct, map, or slice. Maybe not a big deal here, though.